tools/packaging/static-build/initramfs/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+from ubuntu:20.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=America/New_York
```

Comment:
Out of curiosity, why do we need this?

Reply:
During the installation of some required packages, Ubuntu usually installs the tzdata package, it will hangs and waits for user input to continue the installation: https://stackoverflow.com/questions/67452096/docker-build-hangs-based-on-order-of-install

Reply:
Let's set it to UTC / GMT then.
Another possible way to go is simply remove tzdata as already done in the Kata Containers CI.

Reply:
I updated to use UTC, tzdata is defined in docker file but included in the package dependencies.